pkg/appmanager/appManager.go

```diff
@@ -1107,6 +1104,51 @@ func (appMgr *Manager) syncIngresses(
 						}
 					}
 				}
+				if dep.Kind == "URL-Rewrite-Annotation" || dep.Kind == "App-Root-Annotation" {
```

Comment:
Nitpick, but we should probably make all of these dep.Kinds into constants.

Reply:
Can do.